src/components/Dropdown/DropdownMenu/DropdownMenu.tsx

```diff
@@ -0,0 +1,44 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Size } from "@utilities/enum";
+import { ReactElement } from "react";
+import DropdownMenuItem, { MenuItem } from "../DropdownMenuItem/DropdownMenuItem";
+import css from "./DropdownMenu.module.css";
+
+export interface DropdownMenuProps {
+    items: MenuItem[][];
+    onChange: (id: string) => void;
+    size?: Size.Small | Size.Large;
+    activeItemId?: string;
+}
+
+export default function DropdownMenu({
+    items,
+    onChange,
+    size = Size.Small,
+    activeItemId = "",
+}: DropdownMenuProps): ReactElement<DropdownMenuProps> {
+    return (
+        <ul className={`${css.menu} ${size === Size.Large ? css.large : ""}`}>
+            {items.map((menuItems, i) => (
+                <li key={i} className={css.divider} data-test-id="dropdown-divider">
+                    <ul className={css.list} data-test-id="dropdown-item-list">
+                        {menuItems.map(({ id, title, icon, subtitle, size, style, disabled }) => (
+                            <DropdownMenuItem
+                                key={id}
+                                onClick={() => onChange(id)}
+                                title={title}
+                                icon={icon}
+                                subtitle={subtitle}
+                                size={size}
+                                style={style}
+                                disabled={disabled}
+                                active={id === activeItemId}
+                            />
+                        ))}
```

Comment:
Is there a reason behind using object destructuring here for the `menuItem`? I think if we would just use 

```tsx
{menuItems.map((menuItem) => (
    <DropdownMenuItem
        key={menuItem.id}
        onClick={() => onChange(menuItem.id)}
        title={menuItem.title}
        icon={menuItem.icon}
        subtitle={menuItem.subtitle}
        size={menuItem.size}
        style={menuItem.style}
        disabled={menuItem.disabled}
        active={menuItem.id === activeItemId}
    />
))}
```

it would be easier to read and understand where each variable is coming from. Additionally it would help avoiding naming collisions with other variables that are defined in an upper scope. For example when using object destructuring the variable `size` on line 26 is colliding with the variable `size` from line 18.

@NadiaPosch What do you think? :)

Reply:
I try to destructure whenever it's possible to reduce redundancy but if you prefer the version above that's fine for me :)

Reply:
In this case I'd prefer not to destructure the object because we are otherwise overwriting variables from the upper scope.